clang/lib/Frontend/InitPreprocessor.cpp

```diff
@@ -1138,6 +1138,14 @@ static void InitializePredefinedMacros(const TargetInfo &TI,
   if (LangOpts.SYCLUnnamedLambda)
     Builder.defineMacro("__SYCL_UNNAMED_LAMBDA__", "1");
 
+  if (LangOpts.SYCLIsDevice) {
+    // Enable SYCL_DISABLE_PARALLEL_FOR_RANGE_ROUNDING macro for
```

Comment:
I might consider putting this at line 1134 to save the SYCLIsDevice check.

Reply:
Thank you. I will do that.

Reply:
done